#8336 - triggers description (#7)
G1ANT-Robot/G1ANT.Addon.Core: G1ANT.Addon.Core/Triggers/ScheduleTrigger.md

> # ScheduleTrigger

Author: ```suggestion
# Schedule Trigger
```

> ScheduleTrigger executes tasks at a specified time.

Author: ```suggestion
Schedule Trigger executes tasks at a specified time.
```

## Initial Arguments

| Trigger Argument | Required | Default value | Description |
| -------- | ---- | -------- | ------------- |
| `CrontabExpression` | yes |  | Time specified for the trigger to be called at. For more information about setting this crontab argument, please visit [Crontab](https://crontab.guru/) website. |

## Task Arguments

| Trigger Argument | Description |
| -------- | ---- |
| `Time` | Amount of time it took the trigger to perform the action |

> ## Example of Defining a ScheduleTrigger in Settings

Author: ```suggestion
## Example of Defining a Schedule Trigger in Settings
```

![ScheduleTrigger in Settings](https://manual.g1ant.com/link/G1ANT.Manual/-assets/scheduletriggerexample.png)

> ## Example of Defining a ScheduleTrigger in the Config File

Author: ```suggestion
## Example of Defining a Schedule Trigger in the Config File
```
